From 22e504ceb03bb05e38f395afb039f0e0d6628936 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@uclibc-ng.org>
Date: Sat, 19 Aug 2017 07:54:03 +0200
Subject: fix compile issue when DOASSERTS=y

Pthread pid caching was removed recently but an assert is still
present which checks pthread->pid, and this breaks the build when
debugging is enabled.

Reported-By: Bogdan Harjoc <harjoc@gmail.com>
---
 libpthread/nptl/pthread_getattr_np.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libpthread/nptl/pthread_getattr_np.c b/libpthread/nptl/pthread_getattr_np.c
index 6df6c8e..e808539 100644
--- a/libpthread/nptl/pthread_getattr_np.c
+++ b/libpthread/nptl/pthread_getattr_np.c
@@ -66,7 +66,6 @@ pthread_getattr_np (
     {
       /* No stack information available.  This must be for the initial
 	 thread.  Get the info in some magical way.  */
-      assert (abs (thread->pid) == thread->tid);
 
       /* Stack size limit.  */
       struct rlimit rl;
-- 
cgit v0.11.2-3-g2eb4
